### PR TITLE
Make SNS documentation ready for publishing

### PR DESCRIPTION
**Pull request by LaraAS**

### CODE REVIEW

File: docs/developer-docs/integrations/sns/get-sns/preparation.md [hunk 1]
```diff
@@ -52,185 +57,107 @@ tokens at that price.
...
 There are some parameters that you have to actively set and others that are set to a
 default value but that you can also change if you like.
 To make sure that all parameters are set to valid values, that are also consistent
 with each other, you can use a tool that validates your input file. 
 
 **To create the SNS parameter yaml file and validate it, follow the steps
-'Defining the initial parameters for the SNS'.** <!--TODO-CLI/dfx-Link: -->
-<!-- QUESTION: is it important which ID is used to do this? -->
+'Deployment Arguments' in the
+[SNS CLI tool](https://gitlab.com/dfinity-lab/public/ic/-/tree/master/rs/sns/cli).**
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/get-sns/preparation.md:160]:
```suggestion
[SNS CLI tool](https://gitlab.com/dfinity-lab/public/ic/-/tree/master/rs/sns/cli).
The tool is also available through the dfx cache like this: `$(dfx cache show)/sns`**
```

File: docs/developer-docs/integrations/sns/get-sns/preparation.md [hunk 2]
```diff
@@ -306,3 +226,17 @@ decentralization sale. As more of the sale tokens are sold in the future, the
 voting power multiplier increases until it is 1 when all sale tokens have been
 sold.
 
+### SNS decentralization sale parameters
+As mentioned, the parameters of the decentralization sale are 
+set in the NNS proposal that starts the sale.
+
+They included configurations such as the minimum number of
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/get-sns/preparation.md:233]:
```suggestion
They include configurations such as the minimum number of 
```

File: docs/developer-docs/integrations/sns/index.md [hunk 3]
```diff
@@ -1,29 +1,76 @@
...
+or decentralized exchanges.  
 
-<!--## DAOs and Tokenomics documentation-->
+In this documentation you will find
+* [A technical introduction to the SNS](sns-intro.md)
 
-In tokenomics you will find
+* Information on how to **get an SNS**, which includes
+    * [An introduction how to get an SNS](get-sns/get-sns-intro.md) which
+      exlains the high level stages of gettin an SNS
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/index.md:29]:
```suggestion
      explains the high level stages of getting an SNS
```

File: docs/developer-docs/integrations/sns/index.md [hunk 4]
```diff
@@ -1,29 +1,76 @@
...
+    * [Technical preparations for getting an SNS](get-sns/preparation.md) which
+      describes the technical steps required before getting an SNS. here
+      We also link to non-technical steps that we recommend,
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/index.md:32]:
```suggestion
      describes the technical steps required before getting an SNS.
      We also link to non-technical steps that we recommend,
```

File: docs/developer-docs/integrations/sns/index.md [hunk 5]
```diff
@@ -1,29 +1,76 @@
...
+  * [Guidelines how to integrate with the index canister](integrate-sns/index-integration.md)
 
-<!--## User Guides-->
+* Guidelines for how to **manage an SNS**, which includes
+  * [An introduction to managing an SNS](managing-sns/manage-sns-intro.md)
+  * [Tips regarding cycles management for the cansiters](managing-sns/cycles-usage.md)
+  * Information on nervous system parameters that can be configured
+    in each SNS <!--[Information on nervous system parameters that can be configured
+    in each SNS](managing-sns/nervous-system-parameters.md)-->
+  * Information no how SNS are upgraded <!--[Information no how SNS are upgraded](managing-sns/upgradeSNS.md)-->
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/index.md:49]:
```suggestion
  * Information on how SNS are upgraded <!--[Information on how SNS are upgraded](managing-sns/upgradeSNS.md)-->  
```

File: docs/developer-docs/integrations/sns/index.md [hunk 6]
```diff
@@ -1,29 +1,76 @@
...
+    in each SNS <!--[Information on nervous system parameters that can be configured
+    in each SNS](managing-sns/nervous-system-parameters.md)-->
+  * Information no how SNS are upgraded <!--[Information no how SNS are upgraded](managing-sns/upgradeSNS.md)-->  
+  * A guideline for SNS proposals <!--[A guideline for SNS proposals](managing-sns/proposal-guide.md)-->
+ 
+## DAOs and Tokenomics documentation 
+This documentation is less technical.
+It targets for example developers who want to get an SNS and
+have to make certain non-technical decisions too.
+It also targets interestd developers who would like to understand
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/index.md:56]:
```suggestion
It also targets interested developers who would like to understand
```

File: docs/developer-docs/integrations/sns/index.md [hunk 7]
```diff
@@ -1,29 +1,76 @@
...
-On the wiki you will find
+In this documentation you will find
+* [An introduction to DAOs and
+  SNSs](../../../tokenomics/sns/sns-intro-tokens.md)
+* [An introduction to tokenomics of 
+  DAOs](../../../tokenomics/sns/tokenomics.md)
+* [An introduction to SNS 
+  rewards](../../../tokenomics/sns/rewards.md)
+* [Tips on what to consider before getting an
+  SNS](../../../tokenomics/sns/predeployment-considerations.md)
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/index.md:67]:
```suggestion
* [An introduction to DAOs and SNSs](../../../tokenomics/sns/sns-intro-tokens.md)
* [An introduction to tokenomics of DAOs](../../../tokenomics/sns/tokenomics.md)
* [An introduction to SNS rewards](../../../tokenomics/sns/rewards.md)
* [Tips on what to consider before getting an SNS](../../../tokenomics/sns/predeployment-considerations.md)
```
linebreaks break links

File: docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+## The need to observe the canisters' cycles
+:::danger
+An SNS community must ensure that all SNS and dapp canisters
+have sufficient cycles by manually sending cycles when necessary to
+**each individual SNS canister**.
+:::
+
+1. _Manually_ means that someone has to monitor the cycles of the
+   canisters and actively make calls to send additional cycles to them
+   when the canisters' cycles balance get low.
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md:12]:
```suggestion
   when the canisters' cycles balance gets low.
```

File: docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+   
+2. This process has to be repeated for _each indiviudal SNS canister_ 
+and also for _each dapp canister_.
+   
+At least the latter is not new to developers who built a dapp on the IC.
+Nevertheless, we want to underline the importance of this in the context
+of an SNS: If the SNS goverannce or SNS ledger canister run out of cycles
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md:19]:
```suggestion
of an SNS: If the SNS governance or SNS ledger canister run out of cycles
```

File: docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+and get deleted, this is of course critical as users loose their
+liquid and staked tokens and as the dapp canisters cannot be goverend
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md:21]:
```suggestion
liquid and staked tokens and as the dapp canisters cannot be governed
```

File: docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+first initialized.
+
+This specifically means that SNS communities should be aware of the
+following:
+:::danger
+The SNS ledger canister automatically spawns new archive
+canisters.
+When this happens, the cycles balance of these new canisters has
+to be managed. The ledger's cycles balance decreases by a 
+significat amount if a new archive is spawn.
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md:47]:
```suggestion
significat amount if a new archive is spawned.
```

File: docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+* SNS canisters are started with already a large number of cycles,
+  namely 180T
+* the SNS ledger is started with 60T (2*30T) cycles such
+  that when the first archive is spawn,
+  both should have roughly 30T cycles (as have all other SNS
+  canisters when they are installed).
+
+
+## How to observe and manage the canisters' cycles
+We next describe in more detail how you acn monitor and manage the
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md:62]:
```suggestion
We next describe in more detail how you can monitor and manage the
```

File: docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+cycles of the SNS and dapp canisters.
+
+### 1. Find all SNS and governed dapp canisters and their cycle balance.
+SNS root knows about all these canisters and their cycles. 
+You can get all the SNS and associated dapp cansiter's status,
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md:67]:
```suggestion
You can get all the SNS and associated dapp canisters' status,
```

File: docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+low on cycles, you can top up the canister's cycles.
+<!-- as follows
+dfx,
+dashboard?-->
+
+To do this, you need a wallet with sufficient cycles.
+In an SNS, can look for people who donate cycles in this way.
+Otherwise, you can make a proposal that transfers some ICP
+from the SNS treasury to an individual who is 
+trusted to converte these to cycles and make such payments.
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md:83]:
```suggestion
trusted to convert these to cycles and make such payments.
```

File: docs/developer-docs/integrations/sns/managing-sns/manage-sns-intro.md [hunk 15]
```diff
@@ -1 +1,58 @@
...
+a change in the dapp or to modify a parameter in the SNS
+governance and, if the majority agrees, the proposal is
+automatically executed on-chain.
+For this reason, the following pages are not only relevant
+for developers but for any member of an SNS community
+who would like to actively help managing the SNS.
+:::
+
+On this page we will introduce a 
+few sapect that are relevant for _managing_ an SNS.
```
Comment (sesi200) [inline docs/developer-docs/integrations/sns/managing-sns/manage-sns-intro.md:21]:
```suggestion
few aspects that are relevant for _managing_ an SNS.
```

**Review by sesi200 — APPROVED** [review]

File: docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md (hunk 8 above)
Comment (LaraAS) [reply docs/developer-docs/integrations/sns/managing-sns/cycles-usage.md:12]: I think this is plural as it is the cycles balance of multiple canisters.

**Pull request merged**